apps/dapp/src/components/Notification/NotificationManager.tsx

```diff
@@ -1,16 +1,54 @@
-import React from 'react';
 import styled from 'styled-components';
 import { useNotification } from 'providers/NotificationProvider';
 import Notification from 'components/Notification/Notification';
+import { useEffect, useState } from 'react';
+
+const AUTOHIDE_TIMER = 1000 * 10;
 
 const NotificationManager = () => {
   const { notifications } = useNotification();
 
+  const { closeNotification } = useNotification();
+
+  // Array indicating which notifications are open.
+  // I know what you're thinking - "couldn't this just
```

Comment:
appreciate the context, we can cut this extended comment for now 🙂

Reply:
done